doc/zypper.8.txt

```diff
@@ -656,7 +656,11 @@ include::{incdir}/option_Solver_Flags_Installs.txt[]
 	+
 	If no repositories are specified via the *--from* option, zypper will do a global upgrade with all defined repositories. This global form of dup will also consider unchanged installed packages and re-evaluate their dependencies. This can be a problem if the system contains conflicting repositories, like repositories for two different distribution releases. This often happens if one forgets to remove an older release repository after adding a new one, say openSUSE 13.1 and openSUSE 13.2.
 	+
-	For all repositories which have the distribution version within their URL (like https://download.opensuse.org/distribution/'13.1'/repo/oss) using the *$releasever* variable instead may be helpful (https://download.opensuse.org/distribution/*$releasever*/repo/oss). The variable is per default substituted by the current distributions version ('13.1') This value can be overwritten using the *--releasever* global option. Calling *zypper --releasever 13.2*'...' will cause these repos to use the new location (https://download.opensuse.org/distribution/'13.2'/repo/oss) without need to add/remove anything. Once the *dup* is performed, *$releasever* will default to the new distribution version. See section *Repository Management* for more info about variable substitution.
+	For all repositories which have the distribution version within their URL (like https://download.opensuse.org/distribution/'13.1'/repo/oss) using the *$releasever* variable instead may be helpful (https://download.opensuse.org/distribution/*$releasever*/repo/oss). The variable is per default substituted by the current distributions version ('13.1').
+	+
+	This value can be temporarily overwritten in the current zypper command by using the *--releasever* global option. Calling *zypper --releasever 13.2*'...' will cause these repos to use the new location (https://download.opensuse.org/distribution/'13.2'/repo/oss) without need to add/remove anything. But you'll need to use *--releasever 13.2* with every zypper command until the distribution upgrade was actually performed. Once the *dup* is done, *$releasever* will default to the new distribution version '13.2' and *--releasever* is no longer needed.
```

Comment:
without need to add/remove anything. -> without **the** need to add/remove anything.

Reply:
fixed